bluesky/callbacks/mpl_plotting.py

```diff
@@ -336,6 +342,7 @@ def __init__(self, raster_shape, I, *,
         self.im = None
         self.extent = extent
         self.aspect = aspect
+        self.axes_positive = axes_positive
```

Comment:
It might be more readable to unpack this into two private attributes here (and do the validation here).